discovery-server/src/test/java/com/facebook/airlift/discovery/store/TestEntry.java

```diff
@@ -40,14 +40,14 @@ public void testSerialization()
     {
         JsonCodec<Entry> codec = jsonCodec(Entry.class);
 
-        Entry expected = entryOf("fruit", "apple", 123, 456);
+        Entry expected = entryOf("fruit", "apple", 123, 456, 3000L);
```

Comment:
The test passes without the change as well. And when i tried deserialize the object, it returns the right argument.
codec.toJson(expected) output:
```
{
  "key" : "ZnJ1aXQ=",
  "value" : "YXBwbGU=",
  "version" : {
    "sequence" : 123
  },
  "timestamp" : 456,
  "maxAge" : 3000
}
```

So not sure if this is the case. Can you double check and confirm the behavior is broken?

Reply:
I could reproduce your result as well. A little stange here but the issue of non-expiring entries disappeared after we fixed serialize. I am going to reproduce the issue and take a futher debugging later.

Reply:
Please provide more insight on what you are noticing. Is it memory increased on coordinator due to the node not expiring or cluster keep sending traffic to the node even after it's down? Having more insight into the issue might help us to understand what's going on and can point out to areas to look into to address it.

Reply:
We found there were some invalid node entries in discovery server and the maxAge fields of the entries were null. So we changed the serialization name. After that we never saw this kind of entries any more.

Reply:
We deploy multiple coordinators in our cluster. Each coordinator has a embedded discovery server enabled. We implements a ZK-based service inventory to make coordinators find each other.